stonesoup/platform/base.py

```diff
@@ -1,122 +1,83 @@
-# -*- coding: utf-8 -*-
-import datetime
-import weakref
-from abc import abstractmethod, ABC
-from functools import lru_cache
-from math import cos, sin
-from typing import Sequence, MutableSequence, Optional, TYPE_CHECKING
+from typing import MutableSequence
 
-from scipy.linalg import expm
-import numpy as np
+from stonesoup.base import Property, Base
+from stonesoup.moveable import Movable, FixedMovable, MovingMovable, MultiTransitionMovable
+from stonesoup.sensor.sensor import Sensor
 
-from ..functions import cart2sphere, cart2pol, rotz
-from ..types.array import StateVector
-from ..base import Property
-from ..types.state import State, StateMutableSequence
-from ..models.transition import TransitionModel
 
-if TYPE_CHECKING:
-    from ..sensor.base import BaseSensor
-
-
-class Platform(StateMutableSequence, ABC):
+class Platform(Base):
     """A platform that can carry a number of different sensors.
 
-        The location of platform mounted sensors will be maintained relative to
-        the sensor position. Platforms move within a 2 or 3 dimensional
-        rectangular cartesian space.
+    The location of platform mounted sensors will be maintained relative to
+    the sensor position. Platforms move within a 2 or 3 dimensional
+    rectangular cartesian space.
 
-        A simple platform is considered to always be aligned with its principle
-        velocity. It does not take into account issues such as bank angle or body
-        deformation (e.g. flex).
+    A simple platform is considered to always be aligned with its principle
+    velocity. It does not take into account issues such as bank angle or body
+    deformation (e.g. flex).
 
+    Movement is controlled by the Platform's :attr:`Platform.movement_controller`, and access to
+    attributes of the Platform is proxied to the movement controller, to allow the Platform to
+    report it's own position, orientation etc.
 
-        .. note:: This class abstract and not intended to be instantiated. To get the behaviour of
```

Comment:
Probably want to still mention that this is not meant to be instantiated (due to default movement controller being None)

Reply:
This is still in the documentation (current line 26). It is marked as deleted, but then also marked as re-added later (as the text changed, along with the code around it).